pkg/asset/manifests/azure/cluster.go

```diff
@@ -38,6 +40,11 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 	})
 
 	resourceGroup := installConfig.Config.Platform.Azure.ClusterResourceGroupName(clusterID.InfraID)
+	controlPlaneSubnet := installConfig.Config.Platform.Azure.ControlPlaneSubnetName(clusterID.InfraID)
+	controlPlaneNSG := installConfig.Config.Platform.Azure.ControlPlaneNetworkSecurityGroupName(clusterID.InfraID)
+	computeSubnet := installConfig.Config.Platform.Azure.ComputeSubnetName(clusterID.InfraID)
+	computeNSG := installConfig.Config.Platform.Azure.ComputeNetworkSecurityGroupName(clusterID.InfraID)
```

Comment:
Right now we just have a single cluster security group shared between the two subnets. Do we have a motivation for having separate NSGs per subnet?

Reply:
Updated.